checks/check_status.py

```diff
@@ -625,14 +625,29 @@ def __init__(self, queue_length=0, queue_size=0, flush_count=0, transactions_rec
         self.transactions_received = transactions_received
         self.transactions_flushed = transactions_flushed
 
+        self.proxy_data = get_config().get('proxy_settings', None)
+
     def body_lines(self):
         lines = [
             "Queue Size: %s bytes" % self.queue_size,
             "Queue Length: %s" % self.queue_length,
             "Flush Count: %s" % self.flush_count,
             "Transactions received: %s" % self.transactions_received,
-            "Transactions flushed: %s" % self.transactions_flushed
+            "Transactions flushed: %s" % self.transactions_flushed,
+            ""
         ]
+
+        if self.proxy_data:
+            lines += [
+                "Proxy",
+                "=====",
+                "",
+                "  Host: %s" % self.proxy_data.get('host'),
+                "  Port: %s" % self.proxy_data.get('port')
+            ]
+        else:
```

Comment:
Let's not display anything if there is no configured proxy, it would just add useless elements to the info page.